From b3b325faf8e2e344d181a265d1a5fa40fdb7cb30 Mon Sep 17 00:00:00 2001
From: Will Thompson <wjt@endlessos.org>
Date: Wed, 27 Apr 2022 22:02:04 +0100
Subject: [PATCH 06/13] metainfo: Add <releases> stanza

This is mandatory on Flathub.

These notes are taken from
https://github.com/gobby/gobby/releases/tag/v0.6.0 .
---
 gobby-0.5.metainfo.xml | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/gobby-0.5.metainfo.xml b/gobby-0.5.metainfo.xml
index a89d5d8..8d989f7 100644
--- a/gobby-0.5.metainfo.xml
+++ b/gobby-0.5.metainfo.xml
@@ -28,4 +28,22 @@
 
   <url type="homepage">https://gobby.github.io/</url>
   <url type="bugtracker">https://github.com/gobby/gobby/issues</url>
+
+  <releases>
+    <release version="0.6" date="2021-01-31">
+      <description>
+        <ul>
+          <li>Remove support for GTK+ 2.x; at least GTK+ 3.6 is required now.</li>
+          <li>Enable TCP keepalives, so that inactive connections will drop automatically after a short time.</li>
+          <li>Gobby is now registered as a handler for infinote:// URLs.</li>
+          <li>Remember hosts previously connected to and presents them in the list next time Gobby is started until expilictly removed.</li>
+          <li>Gobby can now show connection parameters, in particular TLS parameters used for the connection.</li>
+          <li>Gobby's configuration options are stored in GSettings instead of a custam config.xml file.</li>
+          <li>The option to not set a certificate in the initial welcome dialog is no longer available.</li>
+          <li>Gobby can now use the default system CAs on Windows.</li>
+          <li>Small bug fixes and code base modernization.</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
 </component>
-- 
2.30.2

